Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/resources/static/processdiscoverer/zul/costTable.zul

```diff
@@ -60,6 +60,7 @@
       </label>
       <combobox
           disabled="${arg.viewOnly}"
+          readonly="true"
```

Comment:
Since this box is now readonly, we should also change the currency_hint message to reflect that. The current message says "Choose a currency from the list or type in if not available", implying that a user can type in the box.